specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.1/TextAnalytics.json

```diff
@@ -0,0 +1,1346 @@
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "showStats",
+            "in": "query",
+            "description": "(Optional) if set to true, response will contain input and document level statistics.",
+            "type": "boolean",
+            "required": false
+          },
+          {
+            "in": "body",
+            "name": "input",
+            "description": "Collection of documents to analyze.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/MultiLanguageBatchInput"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A successful call results in a list of recognized entities returned for each valid document.",
+            "schema": {
+              "$ref": "#/definitions/EntitiesResult"
+            }
+          },
+          "default": {
+            "description": "Error Response",
+            "schema": {
+              "$ref": "#/definitions/TextAnalyticsError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Entities request": {
+            "$ref": ".//examples//SuccessfulEntitiesRequest.json"
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/entities/recognition/pii": {
+      "post": {
+        "summary": "Entities containing personal information",
+        "description": "The API returns a list of entities with personal information (\\\"SSN\\\", \\\"Bank Account\\\" etc) in the document. For the list of supported entity types, check <a href=\"https://aka.ms/tanerpii\">Supported Entity Types in Text Analytics API</a>. See the <a href=\"https://aka.ms/talangs\">Supported languages in Text Analytics API</a> for the list of enabled languages.\n",
+        "operationId": "EntitiesRecognitionPii",
+        "consumes": [
+          "application/json",
+          "text/json"
+        ],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "name": "model-version",
+            "in": "query",
+            "description": "(Optional) This value indicates which model will be used for scoring. If a model-version is not specified, the API should default to the latest, non-preview version. ",
+            "type": "string"
+          },
+          {
+            "name": "showStats",
+            "in": "query",
+            "description": "(Optional) if set to true, response will contain input and document level statistics.",
+            "type": "boolean"
+          },
+          {
+            "name": "domain",
+            "in": "query",
+            "description": "(Optional) if set to phi, response will contain only PHI entities.",
```

Comment:
1. Are we allowed to use "phi"? Should we use "medical" instead? 
2. Put phi in quotes `"`

Reply:
have put it in quotes. Will discuss with Ashly regarding the name